jsettlers.logic/src/main/java/jsettlers/logic/map/loading/original/OriginalMapFileContentReader.java

```diff
@@ -363,20 +398,20 @@ public void readBasicMapInformation(int previewWidth, int previewHeight) {
 		int unknown = readBEWordFrom(pos);
 		pos += 2;
 
-		float scaleX = wh / width;
-		float scaleY = wh / height;
+		int scaleX = wh / width;
+		int scaleY = wh / height;
```

Comment:
I don't know. wh and width are integers, so using an int type here does not change anything